andela/travel_tool_front
# #160718924 Add Mobile Responsiveness to the New Request Form

- make the form respond well to mobile screen sizes

#### What does this PR do?
- Adds mobile responsiveness to the New Request Form

#### Description of Task to be completed?
- Ensure that the form appears well on mobile device screens

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-request-form-responsiveness-160718924`
- Install dependencies - `yarn install`
- Start the server  - `make start`
- Navigate to `localhost:3000` in a browser
- Change your role to travel admin in the database then navigate to residence => manage and refresh the page
- Change your role to super admin in the database then navigate to settings => user roles, refresh the page

#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
 #160718924

#### Screenshots (if appropriate)


#### Questions:
